### PR TITLE
fix: Add steps to enable touch on Microtips OLDI on SK-AM62B-P1

### PR DESCRIPTION
**Pull request by jainswamil**
Add steps for how to enable touch on Microtips panel on AM625 sk evm B/-P1 revision

### CODE REVIEW

File: source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst [hunk 1]
```diff
@@ -55,3 +55,25 @@ is supported on platforms:
 		root@am62xx-evm:~# modprobe ili210x
 
 After re-initializing the module, the touch IC should now be registered as an input and touch should be functional.
+
+**Note**: The following steps are only applicable on AM625 SK EVM revision B/B-P1 board
```
Comment (StaticRocket) [inline source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst:59]: This is not the way device specific documentation should be done. There are clear sections in this documentation that are not using the section notation. Notes are not a good way to indicate a platform switch.

File: source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst [hunk 2]
```diff
@@ -55,3 +55,25 @@ is supported on platforms:
...
+
+#. Run the following command
+
+        .. code-block:: bash
```
Comment (StaticRocket) [inline source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst:63]:
```suggestion
        .. code-block:: console
```

File: source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst [hunk 3]
```diff
@@ -55,3 +55,25 @@ is supported on platforms:
...
+
+                root@am62xx-evm:~# modprobe -r ili210x
+
+#. Power off the LVDS panel by disconnecting the 12V barrel jack (do not power off the SK EVM)
+#. Power on the LVDS panel
+#. Run the following command
+
+        .. code-block:: bash
```
Comment (StaticRocket) [inline source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst:71]:
```suggestion
        .. code-block:: console
```

**Review by StaticRocket — CHANGES_REQUESTED** [review]

**Review by cshilwant — COMMENTED** [review]
Nit pick,

docs in title has no meaning. One suggestion for commit message title => "fix: Add steps to enable touch on Microtips OLDI on SK-AM62B-P1"

And then in commit message body go ahead and add more details

File: source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst (hunk 1 above)
Comment (cshilwant) [inline source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst:59]: Better to link

https://www.ti.com/tool/SK-AM62B

https://www.ti.com/tool/SK-AM62B-P1


Similar to how @praneethbajjuri mentions in his PR - https://github.com/TexasInstruments/processor-sdk-doc/pull/77/files#diff-b18514cfd2d8f65bf16517bf5593a255d5417eb1594fe4e1157fd93291f78e06

**Review by cshilwant — CHANGES_REQUESTED** [review]

File: source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst (hunk 1 above)
Comment (jainswamil) [reply source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst:59]: Done

Comment (jainswamil) [reply source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst]: Created sections for different revisions.

File: source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst (hunk 2 above)
Comment (jainswamil) [reply source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst]: Done

File: source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst (hunk 3 above)
Comment (jainswamil) [reply source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst]: Done

File: source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst
Comment (cshilwant) [inline source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst:1]: The indentation of directives like code-block, note, etc doesn't follow the guidelines added in CONTRIBUTING - https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace

Code sections should clearly line up with directive

File: source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst [hunk 4]
```diff
@@ -33,14 +36,16 @@ is supported on platforms:
       +---------------+---------------------------------------------+
 
 
-**Steps to enable Touch on the Microtips Panel on AM625 SK EVM:**
+Steps to enable Touch on the Microtips Panel on AM625 SK EVM
+------------------------------------------------------------
 
-**Note**: The following steps are only applicable on AM625 SK EVM revision E3 board
+E3 revision
```
Comment (cshilwant) [inline source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst:42]:
```suggestion
AM62X SK E3 revision
```

File: source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst [hunk 5]
```diff
@@ -50,8 +55,31 @@ is supported on platforms:
...
 #. Run the following command:
 
-	.. code-block:: bash
+	.. code-block:: console
 
 		root@am62xx-evm:~# modprobe ili210x
 
 After re-initializing the module, the touch IC should now be registered as an input and touch should be functional.
+
+B/B-P1 revision
```
Comment (cshilwant) [inline source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst:64]:
```suggestion
AM62B SK & AM62B-P1
```

Better to have consistency across variants names - https://github.com/TexasInstruments/processor-sdk-doc/pull/105/files#diff-7eb9b090bbfd51a34a333ebfdd6d81739b90943f430fc2aad0185f8596915d00R10

**Review by cshilwant — CHANGES_REQUESTED** [review]

File: source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst
Comment (jainswamil) [reply source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst:1]: Updated

File: source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst (hunk 4 above)
Comment (jainswamil) [reply source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst]: Done

File: source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst (hunk 5 above)
Comment (jainswamil) [reply source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst]: Done

File: source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst [hunk 6]
```diff
@@ -33,25 +36,50 @@ is supported on platforms:
...
+
+   .. code-block:: console
+
+      root@am62xx-evm:~# modprobe ili210x
+
+After re-initializing the module, the touch IC should now be registered as an input and touch should be functional.
+
+.. note::
+
+   ``i2ctransfer`` not required for AM625 SK EVM B/B-P1 revision
```
Comment (cshilwant) [inline source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst:85]: Same comment on consistency as earlier https://github.com/TexasInstruments/processor-sdk-doc/pull/105#discussion_r1881806894

```suggestion
   ``i2ctransfer`` not required for AM62B SK & AM62B-P1
```

**Review by cshilwant — CHANGES_REQUESTED** [review]

File: source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst (hunk 6 above)
Comment (jainswamil) [reply source/linux/How_to_Guides/Target/How_to_enable_Microtips_LVDS_Panel.rst]: Done

**Review by jsuhaas22 — APPROVED** [review]

**Review by StaticRocket — APPROVED** [review]

**Pull request merged**